.vscode/settings.json

```diff
@@ -0,0 +1,35 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "prettier.bracketSpacing": true,
```

Comment:
The use case to enforce this is to enforce a certain style.
Even if people use other editors they can still look at this to understand how we want style to work.